pkg/resmap/resmap.go

```diff
@@ -226,14 +230,25 @@ func MergeWithoutOverride(maps ...ResMap) (ResMap, error) {
 }
 
 // MergeWithOverride combines multiple ResMap instances, allowing and sometimes
-// demanding certain collisions.
+// demanding certain collisions and skipping nil maps.
+// In case if all of the maps are nil, an empty ResMap is returned.
 // When looping over the instances to combine them, if a resource id for resource X
 // is found to be already in the combined map, then the behavior field for X
 // must be BehaviorMerge or BehaviorReplace.  If X is not in the map, then it's
 // behavior cannot be merge or replace.
 func MergeWithOverride(maps ...ResMap) (ResMap, error) {
 	result := maps[0]
+	if result == nil {
+		result = ResMap{}
+	}
 	for _, m := range maps[1:] {
+		if m == nil {
+			continue
+		}
+		if result == nil {
```

Comment:
how can this be true given L242?

Reply:
Oops, leftover from a previous attempt. Removed.